express_relay/sdk/js/src/examples/simpleSearcher.ts

```diff
@@ -75,15 +100,84 @@ class SimpleSearcher {
     }
   }
 
-  async start() {
+  async svmDummyBid() {
+    if (this.privateKeySvm === undefined) {
+      throw new Error("SVM private key not provided");
+    }
+    const secretKey = anchor.utils.bytes.bs58.decode(this.privateKeySvm);
+    const searcher = Keypair.fromSecretKey(secretKey);
+
+    if (argv.svmEndpoint === undefined) {
+      throw new Error("SVM endpoint not provided");
+    }
+    const connection = new Connection(argv.svmEndpoint, "confirmed");
+    const provider = new AnchorProvider(
+      connection,
+      new anchor.Wallet(searcher),
+      {}
+    );
+    const dummy = new Program(dummyIdl as Idl, provider);
+
+    const permission = PublicKey.default;
+    const bidAmount = new anchor.BN(argv.bid);
+
+    const svmConstants = SVM_CONSTANTS[argv.chainId];
+    const dummyPid = DUMMY_PIDS[argv.chainId];
+
+    const ixDummy = await dummy.methods
+      .doNothing()
+      .accountsPartial({
+        payer: searcher.publicKey,
+        expressRelay: svmConstants.expressRelayProgram,
+        sysvarInstructions: anchor.web3.SYSVAR_INSTRUCTIONS_PUBKEY,
+        permission,
+        protocol: dummyPid,
+      })
+      .instruction();
+    ixDummy.programId = dummyPid;
+
+    const txRaw = new anchor.web3.Transaction().add(ixDummy);
+
+    const txPermissioned = await this.client.constructPermissionedTxFromSvmTx(
+      txRaw,
+      dummy.programId,
+      permission,
+      bidAmount,
+      new anchor.BN(Math.round(Date.now() / 1000 + DAY_IN_SECONDS)),
+      secretKey,
+      argv.chainId
+    );
+
     try {
-      await this.client.subscribeChains([argv.chainId]);
-      console.log(
-        `Subscribed to chain ${argv.chainId}. Waiting for opportunities...`
+      const bidId = await this.client.signAndSubmitSvmBid(
+        txPermissioned,
+        permission,
+        bidAmount,
+        [secretKey],
+        argv.chainId
       );
+      console.log(`Successful bid. Bid id ${bidId}`);
     } catch (error) {
-      console.error(error);
-      this.client.websocket?.close();
+      console.error(`Failed to bid: ${error}`);
+    }
+  }
+
+  async start() {
```

Comment:
looking at the logic split between svm and evm I suggest creating a separate simpleSvmSearcher.ts file. The current format makes it a bit confusing to follow what's happening based on chains